clients/VKontakte.php

```diff
@@ -85,7 +85,10 @@ protected function initUserAttributes()
         $response = $this->api('users.get.json', 'GET', [
             'fields' => implode(',', $this->attributeNames),
         ]);
-        $attributes = array_shift($response['response']);
+        $attributes = [];
+        if (!empty($response['response'])) {
+            $attributes = array_shift($response['response']);
+        }
```

Comment:
This does not seem to be correct: if there is no response there are no attributes, which means method `initUserAttributes()` failed and thus should throw an exception or trigger an error, why should it silently fail?